listenbrainz/influx-writer/influx-writer.py

```diff
@@ -169,11 +172,13 @@ def write(self, listen_dicts):
         # remove duplicates on the basis of timestamps
         for user_name in users:
 
-            min_time = users[user_name]['min_time']
-            max_time = users[user_name]['max_time']
+            # get the range of time that we need to get from influx for
+            # deduplication of listens
+            min_time = users[user_name]['min_time'] - TIMESTAMP_DUPLICATE_DIFF
+            max_time = users[user_name]['max_time'] + TIMESTAMP_DUPLICATE_DIFF
 
             # quering for artist name here, since a field must be included in the query.
-            query = """SELECT time, artist_name
+            query = """SELECT time, artist_msid, recording_msid
                          FROM %s
                         WHERE time >= %d000000000
```

Comment:
can we make this more understandable by having a function like `timestamp_to_influx_crappy_nanoseconds` which we call like `get_escaped_measurement_name`?